md2man/roff.go

```diff
@@ -11,7 +11,7 @@ import (
 
 type roffRenderer struct{}
 
-var listCounter int
```

Comment:
oh wow, I can't believe this global is even here... should be moved to the renderer object